_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,133 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Notifier can send a variety of notifications to alert you
+of Forseti events. You can configure notifications to send to different
+channels and in different formats.
+
+### Notification Types
+
+  1. Inventory summary: Count of the resources added in the latest inventory crawl.
+  1. Violations: Individual violations that have been found from the latest scanner run. 
+  1. Cloud Security Command Center (Cloud SCC) findings: Violations converted to Cloud SCC findings format, that's ingestable by Cloud SCC.
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. Cloud Storage
+
+### Notification Formats
+
+  1. Human-readable data: CSV
+  1. Structured data: JSON
+
+## Configuring Notifier
+
+### Inventory Summary
+
+This is a count of what resources have been crawled into Inventory,
+and outputted to Cloud Storage bucket.
+
+```yaml
+notifier:
+    inventory:
+      summary:
+        enabled: true
+        data_format: csv
+        gcs_path: gs://path_to_foo_bucket
+```
+
+  `true` enables the notification, and `false` disables the notification.
+* `data_format`: either `csv` or `json`
+* `gcs_path`: The Cloud Storage bucket to upload the inventory summary.
+
+
+### Violation Notifications
+
+To select how you want Notifier to send violation notifications,
+follow the steps below:
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+On a per-resources basis, the following options are available. You can use
+any combination of notifiers for each resource.
+
+* `should_notify`: Whether a violation for each resource should be sent.
+  `true` enables the notification, and `false` disables the notification.
+
+* `name`: The name of the notifier you want for each resource. You can specify
+  multiple notifiers for each resource.
+
+  The name must match the actual module name for each notifier in 
+  `forseti-security/google/cloud/forseti/notifier/notifiers`,
```

Comment:
Just link to the notifiers directory for now so people can see a list of them.

Reply:
Done